### ISSUE TITLE
can namer ignore directories prepended with _UNPACK_ ?

### ISSUE THREAD

**Issue by PAHelper2** [ThePornDatabase/namer#3]
Hi,

I'm trying to have sabnzbd write downloads to a directory that namer watches.  Part of the process is sabnzbd uses a temp _UNPACK_ directory, with the dir name.  namer sees that directory, waits 30 seconds, and then copies it to the work dir.

This is before sabnzbd is done unpacking, as you can see from this sabnzbd error: 
```
 Error renaming "/namer_watch/_UNPACK_PervMom.22.02.27.Jenna.Noelle.Taste.Of.Jenna.XXX.MP4-P0RNL0V3RSD" to "/namer_watch/PervMom.22.02.27.Jenna.Noelle.Taste.Of.Jenna.XXX.MP4-P0RNL0V3RSD"
```

Can namer somehow be configured to ignore _UNPACK_ directories and only process when they've been renamed?

Thanks!

**Comment by PAHelper2** [comment]
For reference, I also tried disabling the 'Direct Unpack' feature in sabnzbd, but that didn't help at all.

**Comment by 4c0d3r** [comment]
Fixed in 1.0.1, there is now a ignore_dir_regex option in namer.cfg that defaults to .*_UNPACK_.*

**Comment by PAHelper2** [comment]
Just for the record, I can confirm that this works.  (See below.)  Thank you!

```
namer    | Watchdog Config:
namer    |   ignored_dir_regex: .*_UNPACK_.*
namer    |   min_file_size: 100mb
namer    |   del_other_files: False
namer    |   new_relative_path_name: {site}-{date}-{name}/{site}-{date}-{name}-{performers}.{ext}
namer    |   watch_dir: /data/namer_watch
namer    |   work_dir: /data/namer_work
namer    |   failed_dir: /data/namer_failed
namer    |   dest_dir: /data/Scene
namer    |   retry_time: 03:29
namer    |   extra_sleep_time: 60
namer    | 
namer    | Built on: 2022-03-29T01:24:42Z
namer    | Git Hash: 9a1292ad6499c2f9cadfe36a74ddb1b07186b91d
namer    | 2022-03-28 19:04:30 - Start porndb scene watcher.... watching: /data/namer_watch
namer    | 2022-03-28 19:06:29 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:30 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:31 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:32 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:33 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:34 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:35 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:36 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:37 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:38 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:39 - watchdog process called for _UNPACK_PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:41 - watchdog process called for PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD.mp4
namer    | 2022-03-28 19:06:41 - Moving /data/namer_watch/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD to /data/namer_work/PervMom.22.01.16.Penny.Barber.Strict.But.Fair.XXX.MP4-P0RNL0V3RSD for processing
namer    | 2022-03-28 19
```
